src/gui/qgsattributeeditorcontext.h

```diff
@@ -38,6 +38,18 @@ class GUI_EXPORT QgsAttributeEditorContext
 {
   public:
 
+    //! Form modes
+    enum Mode
```

Comment:
This is copied from somewhere, correct? Would it be possible to replace the other one with this one in the long run and deprecate it for QGIS 4?

Reply:
This should be removed again. I forgot after changing back saving the Mode as `QString` here. Another easy to change possibility would be:
- keep this enum here, work here with enum instead of string for mAttributeFormMode
- use in the `QgsAttributeForm` the enums from here
- implement function `modeString` function here instead of `QgsAttributeForm` and remove there.

What should be prefered?